src/orquestra/vqa/utils/__init__.py

```diff
@@ -0,0 +1,20 @@
+from ._openfermion_adapter import openfermion_adapter
```

Comment:
I think that it would make sense to rename it from `utils` to `openfermion`.
If we keep `utils` directory, people will keep adding more and more things here.
And these are all openfermion-related things.
WDYT?